src/analyser/dataOperator1D.cpp

```diff
@@ -1,26 +1,36 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 // Copyright (c) 2024 Team Dissolve and contributors
 
-#include "analyser/dataNormaliser1D.h"
+#include "analyser/dataOperator1D.h"
 #include "math/data1D.h"
 #include "math/integrator.h"
 
-DataNormaliser1D::DataNormaliser1D(Data1D &targetData) : DataNormaliserBase<Data1D, NormalisationFunction1D>(targetData) {}
+DataOperator1D::DataOperator1D(Data1D &targetData) : DataOperatorBase<Data1D, OperateFunction1D>(targetData) {}
 
-void DataNormaliser1D::normalise(NormalisationFunction1D normalisationFunction)
+/*
+ * Data Operation Functions
+ */
+
+// Generic operate function
+void DataOperator1D::operate(OperateFunction1D operateFunction)
```

Comment:
I think just calling the function argument `operator` feels a lot cleaner:
```suggestion
void DataOperator1D::operate(OperateFunction1D operator)
```